remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties

```diff
@@ -13,12 +13,33 @@ remoting.worker-task-max-threads=The maximum number of threads for the remoting
+  and the selected SASL mechanism demands a user name.
+remoting.auth-realm=The authentication realm to use if no authentication {@code CallbackHandler} is specified.
+remoting.sasl-protocol=Where a SaslServer or SaslClient are created by default the protocol specified it 'remoting', this can be used to override this.
+remoting.max-outbound-message-size=The maximum outbound message size to send.  No messages larger than this well be transmitted; attempting to do so will cause an exception on the writing side.
+remoting.buffer-region-size=The size of allocated buffer regions.
+remoting.receive-buffer-size=The size of the largest buffer that this endpoint will accept over a connection.
+remoting.authentication-retries=Specify the number of times a client is allowed to retry authentication before closing the connection.
+remoting.transmit-window-size=The maximum window size of the transmit direction for connection channels, in bytes.
+remoting.max-outbound-messages=The maximum number of concurrent outbound messages on a channel.
+remoting.send-buffer-size=The size of the largest buffer that this endpoint will transmit over a connection.
+remoting.max-inbound-messages=The maximum number of concurrent inbound messages on a channel.
+remoting.receive-window-size=The maximum window size of the receive direction for connection channels, in bytes.
+remoting.heartbeat-interval=The interval to use for connection heartbeat, in milliseconds.  If the connection is idle in the outbound direction\
+  for this amount of time, a ping message will be sent, which will trigger a corresponding reply message.
+remoting.max-inbound-message-size=The maximum inbound message size to be allowed.  Messages exceeding this size will cause an exception to be thrown on the reading side as well as the writing side.
+remoting.max-outbound-channels=The maximum number of outbound channels to support for a connection.
+remoting.max-inbound-channels=The maximum number of inbound channels to support for a connection.
+remoting.server-name=The server side of the connection passes it's name to the client in the initial greeting, by default the name is automatically discovered from the local address of the connection or it can be overridden using this.
+remoting.worker=Worker to use
 
 remoting.connector=The remoting connectors.
 remoting.configuration=Configuration options
 endpoint=Endpoint configuration
 endpoint.add=Adds endpoint
 endpoint.remove=Removes endpoint configuration
+endpoint.deprecated=The child resource for configuring the remoting endpoint is deprecated. Use the attributes on the parent resource to configure the remoting endpoint.
 endpoint.authorize-id=The SASL authorization ID.  Used as authentication user name to use if no authentication {@code CallbackHandler} is specified\
   and the selected SASL mechanism demands a user name.
 endpoint.auth-realm=The authentication realm to use if no authentication {@code CallbackHandler} is specified.
```

Comment:
Looks like there are other places not associated with this change the use `{@code..}` as well.